src/lib/serviceRoutines/statisticsTreat.cpp

```diff
@@ -501,56 +383,18 @@ std::string statisticsTreat
   char  listBuffer[1024];
   bool  reqSemTaken;
 
+  // FIXME P10: should this be using the cacheSemTake()/cacheSemGive() functions ?
```

Comment:
Ok, I'll do that.

Is this relevant enough to mark the "fix" in the CNR? Probably it isn't but... not sure.